web/src/routes.rs

```diff
@@ -13,5 +15,10 @@ pub fn routes(app_state: AppState) -> axum::Router {
         .route("/login", get(users::login::get).post(users::login::post))
         .route("/signup", get(users::signup::get).post(users::signup::post))
         .route("/logout", get(users::login::logout))
+        .route(
+            "/insert_articles",
```

Comment:
Reminder that this got changed to `/articles` and needs to be updated here too (or may when you rebase not positive)

[`fff0454` (#17)](https://github.com/coreyja/knowledge/pull/17/commits/fff04544b6e546dc7c0259d662a8f57e194b9620)